index.html

```diff
@@ -1,10 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
-<head> 
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- google fonts link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="index.css" />
+
     <title>Progress Tracker</title>
-    <link rel="stylesheet" href="/index.css">
-</head>
-<body>
-    <h1>Scrimba Bootcamp Progress Tracker</h1>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <!-- header goes here -->
+    <div class="hero_section">
+      <div class="hero_container">
+        <div class="content">
+          <h1 class="hero_title">Scrimba Bootcamp Progress Tracker</h1>
+          <hr>
+          <p class="hero_description">Welcome! This open source project is for Scrimba Bootcamp students to get experience working with other developers, becoming familiar with the git workflow, and learning best practices for collaborative projects.</p>
+          <button>See the source code </button>
```

Comment:
@rabin-thami this element should be an anchor tag with an `href` attribute linking to our repository. You can style it the same way as you would a button, but the anchor tag is the correct semantic element to use in this case.